int32 selectedUAV
int32 numberofWPs
int32[] wpNumber
float64[] wpX
float64[] wpY
float64[] wpZ
float64[] wpW
float64[] wpLT
